### ISSUE TITLE
error in node2vec fit

### ISSUE THREAD

**Issue by BSharmi** [eliorc/node2vec#60]
Hello,

I ran the commands using the default data in the package and get the following error in the embed node step in the `node2vec.fit` step

`import networkx as nx`
`from node2vec import Node2Vec`
`graph = nx.fast_gnp_random_graph(n=100, p=0.5)`
`node2vec = Node2Vec(graph, dimensions=64, walk_length=30, num_walks=200, workers=4) `
`model = node2vec.fit(window=10, min_count=1, batch_words=4) `

error

`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-66dd459dbca1> in <module>
      9 
     10 # Embed nodes
---> 11 model = node2vec.fit(window=10, min_count=1, batch_words=4)  
~/anaconda3/envs/default/lib/python3.8/site-packages/node2vec/node2vec.py in fit(self, **skip_gram_params)
    181             skip_gram_params['sg'] = 1
    182 
--> 183         return gensim.models.Word2Vec(self.walks, **skip_gram_params)
TypeError: __init__() got an unexpected keyword argument 'size'`

I have installed the required packages. Is there any step I missed?

Would appreciate a response!

Thank you!

**Comment by adamdingliang** [comment]
I had the same error. Waiting for a response!

**Comment by eliorc** [comment]
Yes, gensim has changed their interface, you can wither downgrade gensim to version less than 4 or wait for the weekedn and I'll fix it as I noted in #59

**Comment by adamdingliang** [comment]
Downgrade gensim to 3.8.3 works.

**Comment by BSharmi** [comment]
I would wait for the fix. Thanks very much for looking into it!

**Comment by eliorc** [comment]
Solved in 4.2.0

**Comment by p-dre** [comment]
TypeError: Word2Vec.__init__() got an unexpected keyword argument 'size'

with node2vec  4.6

**Comment by eliorc** [comment]
Can you tell me which `gensim` version is installed?

> TypeError: Word2Vec.**init**() got an unexpected keyword argument 'size'
> 
> with node2vec 4.6

**Comment by MikeB2019x** [comment]
Fix for that is as follows:
replace the variable names in the Word2Vec call: 'iter' -> 'epochs' and 'size' -> 'vector_size'

**Comment by MikeB2019x** [comment]
Sorry about that :-D  I was using: gensim 4.2.0 but encountered the problem while working with the struc2vec repo [here](https://github.com/leoribeiro/struc2vec/pull/12).  Basically a fork of your latest and then tried to get everything working in Python3 with the pr from @steenfatt.

**Comment by eliorc** [comment]
I'll try to check it out in the weekend

**Comment by eliorc** [comment]
I can't find time to do this, I'd appreciate if anyone wants to contribute through a PR @MikeB2019x @p-dre

**Comment by MikeB2019x** [comment]
I can do that.

**Comment by eliorc** [comment]
@MikeB2019x I'd love to accept it!
